coderd/database/queries/aibridge.sql

```diff
@@ -126,8 +128,10 @@ FROM
 JOIN
 	visible_users ON visible_users.id = aibridge_interceptions.initiator_id
 WHERE
+	-- Remove inflight interceptions that lack an ended_at value.
+	aibridge_interceptions.ended_at IS NOT NULL
```

Comment:
I'm not familiar with AI Bridge part of coderd, but shouldn't we cover this change with test cases? cc @dannykopping

Reply:
Absolutely.